packages/embed/README.md

```diff
@@ -2,7 +2,108 @@
 
 Core library written in Typescript. Compiles to vanilla JavaScript.
 
-## Development
+## Usage
+
+### As NPM package
+
+Install:
+
+```shell
+yarn add @typeform/embed@next
+```
+
+Import the lib, CSS and create your embed:
+
+```javascript
+import { createWidget } from '@typeform/embed@next'
```

Comment:
provide complete import?
`{ createWidget, createPopup, createSidebar, ..}`

Reply:
There might be more to come in future. This is supposed to be just a quick and short example. Full list is in `Embed types` below.